app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/newdevicenotice/NewDeviceNoticeEmailAccessScreen.kt

```diff
@@ -0,0 +1,199 @@
+            onEmailAccessToggleChanged = remember(viewModel) {
+                { newState ->
+                    viewModel.trySendAction(EmailAccessToggle(newState = newState))
+                }
+            },
+            onContinueClick = { viewModel.trySendAction(ContinueClick) },
+        )
+    }
+}
+
+@Composable
+private fun NewDeviceNoticeEmailAccessContent(
+    email: String,
+    isEmailAccessEnabled: Boolean,
+    onEmailAccessToggleChanged: (Boolean) -> Unit = {},
+    onContinueClick: () -> Unit = {},
+) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center,
+        modifier = Modifier
+            .fillMaxSize()
+            .verticalScroll(state = rememberScrollState()),
+    ) {
+        HeaderContent()
+        MainContent(
+            email = email,
+            isEmailAccessEnabled = isEmailAccessEnabled,
+            onEmailAccessToggleChanged = onEmailAccessToggleChanged,
+        )
+        BitwardenFilledButton(
+            label = stringResource(R.string.continue_text),
+            onClick = onContinueClick,
+            modifier = Modifier
+                .padding(24.dp)
+                .fillMaxWidth()
+                .imePadding(),
+        )
+        Spacer(modifier = Modifier.navigationBarsPadding())
+    }
+}
+
+/**
+ * Header content containing the warning icon and title.
+ */
+
+@Suppress("MaxLineLength")
+@Composable
+private fun HeaderContent() {
+    Image(
+        painter = rememberVectorPainter(id = R.drawable.warning),
+        contentDescription = null,
+        modifier = Modifier.size(120.dp),
+    )
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.padding(horizontal = 24.dp),
+    ) {
+        Text(
+            text = stringResource(R.string.important_notice),
+            style = BitwardenTheme.typography.headlineMedium,
+            color = BitwardenTheme.colorScheme.text.primary,
+            textAlign = TextAlign.Center,
+        )
+        Spacer(modifier = Modifier.height(8.dp))
+        Text(
+            text = stringResource(
+                R.string.bitwarden_will_soon_send_a_code_to_your_account_email_to_verify_logins_from_new_devices,
+            ),
+            style = BitwardenTheme.typography.bodyLarge,
+            color = BitwardenTheme.colorScheme.text.primary,
+            textAlign = TextAlign.Center,
+        )
+    }
+}
+
+/**
+ * The main content of the screen.
+ */
+@Composable
+private fun MainContent(
+    email: String,
+    isEmailAccessEnabled: Boolean = false,
+    onEmailAccessToggleChanged: (Boolean) -> Unit = {},
+) {
+    Spacer(modifier = Modifier.size(24.dp))
+    Column(
+        modifier = Modifier
+            .padding(horizontal = 22.dp)
+            .fillMaxWidth()
+            .clip(RoundedCornerShape(size = 4.dp))
+            .background(BitwardenTheme.colorScheme.background.secondary),
```

Comment:
We should not be using custom stuff like this, we already have `BitwardenContentCard` to handle stuff like this.

Reply:
The `BitwardenContentCard` didn't seem to adapt to what I wanted.

Reply:
That seems to indicate that the UI is non-standard, I don't think we should be doing that.